testing/src/com/google/idea/testing/runfiles/Runfiles.java

```diff
@@ -32,6 +33,12 @@ public static Path runfilesPath() {
     return runfilesPath("");
   }
 
+  /** Returns the runtime location of a data dependency in an external workspace. */
+  public static Path runfilesPath(String workspace, String path) throws IOException {
+    String combinedPath = workspace + "/" + path;
```

Comment:
I wonder if this would work on Windows.

Reply:
Should work, as far as I know you can use both (`/` and `\`) on Windows as path separators. But if this wasn't a pick I'd also prefer `Path.of` here.